### PR TITLE
fixing facts for mysql server

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY

Addding missing tags support for MySQL server

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
azure_rm_mysqlserver
azure_rm_mysqlserver_facts

##### ANSIBLE VERSION
2.7

### CODE REVIEW

File: test/integration/targets/azure_rm_mysqlserver/tasks/main.yml [hunk 1]
```diff
@@ -76,11 +76,29 @@
     enforce_ssl: True
     admin_username: zimxyz
     admin_password: Testpasswordxyz12!
+    tags:
+      aaa: bbb
+
+- name: Create second instance of MySQL Server
```
Comment (yungezz) [inline test/integration/targets/azure_rm_mysqlserver/tasks/main.yml:82]: delete it also with async

File: test/integration/targets/azure_rm_mysqlserver/tasks/main.yml [hunk 2]
```diff
@@ -276,3 +297,14 @@
...
     that:
       - output.changed == false
+
+- name: Delete second instance of MySQL Server
+  azure_rm_mysqlserver:
+    resource_group: "{{ resource_group }}second"
+    name: mysqlsrv{{ rpfx }}
+    state: absent
+  register: output
+- name: Assert the state has changed
```
Comment (yungezz) [inline test/integration/targets/azure_rm_mysqlserver/tasks/main.yml]: you don't need to wait till deletion done, using async mode to reduce test execution

File: test/integration/targets/azure_rm_mysqlserver/tasks/main.yml (hunk 1 above)
Comment (zikalino) [reply test/integration/targets/azure_rm_mysqlserver/tasks/main.yml:82]: now done

File: test/integration/targets/azure_rm_mysqlserver/tasks/main.yml [hunk 3]
```diff
@@ -276,3 +297,11 @@
...
     that:
       - output.changed == false
+
+- name: Delete second instance of MySQL Server
+  azure_rm_mysqlserver:
+    resource_group: "{{ resource_group }}"
+    name: mysqlsrv{{ rpfx }}second
+    state: absent
+  async: 400
+  pool: 0
```
Comment (yungezz) [inline test/integration/targets/azure_rm_mysqlserver/tasks/main.yml]: poll

**Review by yungezz — APPROVED** [review]

**Pull request merged**